Sources/Mantis/Helpers/Orientation.swift

```diff
@@ -7,70 +7,61 @@
 import UIKit
 
 public struct Orientation {
-    public static var orientation: UIInterfaceOrientation {
-        get {
-            if #available(iOS 13, macOS 10.13, *) {
-                return (UIApplication.shared.windows.first?.windowScene?.interfaceOrientation)!
-            } else {
-                return UIApplication.shared.statusBarOrientation
-            }
+    
+    static var interfaceOrientation: UIInterfaceOrientation {
+        if #available(iOS 13, macOS 10.13, *) {
+            return (application.windows.first?.windowScene?.interfaceOrientation)!
+        } else {
+            return application.statusBarOrientation
         }
     }
     
-    // indicate current device is in the LandScape orientation
-    public static var isLandscape: Bool {
-        get {
-            if #available(iOS 13, macOS 10.13, *) {
-                return UIDevice.current.orientation.isValidInterfaceOrientation
-                    ? UIDevice.current.orientation.isLandscape
-                    : (UIApplication.shared.windows.first?.windowScene?.interfaceOrientation.isLandscape)!
-            } else {
-                return UIDevice.current.orientation.isValidInterfaceOrientation
-                    ? UIDevice.current.orientation.isLandscape
-                    : UIApplication.shared.statusBarOrientation.isLandscape
-            }
-        }
+    static var deviceOrientation: UIDeviceOrientation? {
+        device.orientation.isValidInterfaceOrientation
+            ? device.orientation
+            : nil
     }
-    // indicate current device is in the Portrait orientation
-    public static var isPortrait: Bool {
-        get {
-            if #available(iOS 13, macOS 10.13, *) {
-                return UIDevice.current.orientation.isValidInterfaceOrientation
-                    ? UIDevice.current.orientation.isPortrait
-                    : (UIApplication.shared.windows.first?.windowScene?.interfaceOrientation.isPortrait)!
-            } else {
-                return UIDevice.current.orientation.isValidInterfaceOrientation
-                    ? UIDevice.current.orientation.isPortrait
-                    : UIApplication.shared.statusBarOrientation.isPortrait
-            }
-        }
+    
+    
+    private static var application: UIApplication { Mantis.Config.application }
```

Comment:
Thanks for the PR. The refactor looks good but I don't think we need an `application` property in Mantis.Config.
It seems good enough when you use 
```Swift
private static var application: UIApplication { .shared }
```

Reply:
Yeah, this was added when I continued on the Xcode 13 beta 3 fixes after the cleanup, to get Mantis to compile for Xcode 13. I didn't think ahead, that this commit would also be included in the PR 🙃

In yesterday's release of Xcode 13 b5, they reverted the change that required this property, so I'll just revert this change as well and ping you when you only have the Orientation refactor.